main.go

```diff
@@ -220,6 +223,7 @@ func main() {
 			huh.NewText().
 				Value(&newCommit.Body).
 				Title("Body").
+				CharLimit(0).
```

Comment:
I added this because for some reason I hit a char limit while writing my commit to this PR, setting this explicitly to `0` fixed that

Reply:
Good catch! I hadn't run into that limit before